src/lib.rs

```diff
@@ -61,6 +61,24 @@ enum EmbeddingsWrap {
     View(Embeddings<VocabWrap, StorageViewWrap>),
 }
 
+impl EmbeddingsWrap {
+    fn vocab(&self) -> &VocabWrap {
```

Comment:
Sorry for nitpicking: these can be in alphabetic order.

Reply:
Changed the order